File: BOJ/CLASS 2/1259.js
// 팰린드롬수

const fs = require('fs');
const input = fs.readFileSync('예제.txt').toString().trim().split('\n');

function solution(arr) {
  let answer;

  arr.forEach((x) => {
    answer = 'yes';
    if (x !== '0') {
      let y = x.split('').reverse().join('');

      x = Number(x);
      y = Number(y);

      if (x !== y) answer = 'no';
      console.log(answer);
    } else {
      return;
    }
  });
}

solution(input);

File: BOJ/CLASS 2/Python/1259.py
# 팰린드롬수
while True:
    num = input()

    if num == '0': break

    # r_num = int(''.join(reversed(str(num))))
    # print('yes' if num == r_num else 'no')

    # reversed 대신 슬라이싱 활용
    print('yes' if num == num[::-1] else 'no')